plugins/nf-validation/src/testResources/test_schema.json

```diff
@@ -0,0 +1,142 @@
+            "properties": {
+                "max_cpus": {
+                    "type": "integer",
+                    "description": "Maximum number of CPUs that can be requested for any single job.",
+                    "default": 16,
+                    "fa_icon": "fas fa-microchip",
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the CPU requirement for each process. Should be an integer e.g. `--max_cpus 1`"
+                },
+                "max_memory": {
+                    "type": "string",
+                    "description": "Maximum amount of memory that can be requested for any single job.",
+                    "default": "128.GB",
+                    "fa_icon": "fas fa-memory",
+                    "pattern": "^[\\d\\.]+\\s*.(K|M|G|T)?B$",
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the memory requirement for each process. Should be a string in the format integer-unit e.g. `--max_memory '8.GB'`"
+                },
+                "max_time": {
+                    "type": "string",
+                    "description": "Maximum amount of time that can be requested for any single job.",
+                    "default": "240.h",
+                    "fa_icon": "far fa-clock",
+                    "hidden": true,
+                    "pattern": "^[\\d\\.]+\\.*(s|m|h|d)$",
+                    "help_text": "Use to set an upper-limit for the time requirement for each process. Should be a string in the format integer-unit e.g. `--max_time '2.h'`"
+                }
+            }   
+        },
+        "institutional_config_options": {
+            "title": "Institutional config options",
+            "type": "object",
+            "fa_icon": "fas fa-university",
+            "description": "Parameters used to describe centralised config profiles. These should not be edited.",
+            "help_text": "The centralised nf-core configuration profiles use a handful of pipeline parameters to describe themselves. This information is then printed to the Nextflow log when you run a pipeline. You should not need to change these values when you run a pipeline.",
+            "properties": {}
+        }
+    },
+    "allOf": [
+        {
+        "$ref": "#/definitions/input_output_options"
+        },
+        {
+        "$ref": "#/definitions/reference_genome_options"
+        },
+        {
+        "$ref": "#/definitions/generic_options"
+        },
+        {
+        "$ref": "#/definitions/max_job_request_options"
+        },
+        {
+        "$ref": "#/definitions/institutional_config_options"
+        }
+    ],
+    "properties": {
+        "reads": {
+            "type": "string"
+        },
+        "transcriptome": {
+            "type": "string"
+        },
+        "multiqc": {
+            "type": "string"
+        }
+    }
```

Comment:
Are these `properties` intentionally duplicated here? They are present for each of these parameters in the group too.